# Doctoral dissertation and baccalaureate thesis

Copyright © 2009–2018 Xiaodi Wu


### Classical dendritic cell development excludes alternative myeloid cell fates characterized by distinct transcription factors

Adviser: Kenneth M. Murphy, M.D., Ph.D.  
Submitted at Washington University in St. Louis, June 2016

Dendritic cells (DCs) play key roles in the interface between innate and adaptive immunity. They are capable of detecting a wide range of pathogens and other stimuli; they internalize, process, and present antigens; and they modulate the activity of other immune cells. Recent advances have demonstrated that distinct subsets of DCs play unique roles at steady state and under inflammatory conditions. We found that the transcription factors Bcl11a and Zeb2 are each essential for development of plasmacytoid DCs but dispensable for development of classical DCs _in vivo_. Zeb2-deficient bone marrow cells are also deficient in replenishment of monocytes, which along with their macrophage progeny represent a hematopoietic lineage closely related to that of DCs. Finally, we developed a lineage-tracing mouse strain using Cre recombinase under the control of the transcription factor–encoding locus _Mafb_; with that model, we determined that Langerhans cells in the skin are unique in displaying certain characteristics of both macrophages and DCs.


### Automated genome interpretation as a utility to prioritize variants for clinical and statistical follow-up

Adviser: George M. Church, Ph.D.  
Submitted at Harvard University, March 2009

Recent developments in genome sequencing technology have moved closer to ushering in an era of unprecedented amounts of genetic information. However, data linking genetic variants to traits are currently quite incomplete, and efforts to interpret the four currently published genomes have not yielded many results of note. In this project, I present an interpretation utility that draws upon several data sources and a substitution matrix-based predictive algorithm to prioritize variants for follow-up. Application of this utility to existing genomes replicates phenotypes claimed by previous authors but also casts doubt on their significance, in addition to suggesting additional phenotypes. While demonstrating weaknesses in the underlying data, these results are a step forward in personal genome interpretation and represent part of an evolving bioinformatic approach to extracting functional predictions from large sets of genetic variants.
